.circleci/config.yml

```diff
@@ -129,6 +129,55 @@ jobs:
             - "<<parameters.filepath>>.filtered"
             - "<<parameters.filepath>>.filtered.errlog"
 
+  retag-trivy-dbs:
```

Comment:
This can be re-used for any image, maybe it could have another name

Reply:
Why not just call `retag-registry` with a yaml that contains the DB repositories?

Reply:
I tried it, but it requires way too much work to validate the files and it's too prone to fail, so I reverted to this approach which runs faster and doesn't require as many moving pieces.